lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
See the new commit, if you don't change types for CLOB stored values when a `null` is issued, you'll get that error. We're not using descriptors for CLOB in doctrine. My gist illustrate the fact that:

- by using CLOB with a correct descriptor, you can bind `null` values and everything works
- without descriptor, you can't bind `null` and so you have to use an empty string

Maybe that's not how it's supposed to work, but it is.